Assets/prefabs/GameManager.prefab

```diff
@@ -0,0 +1,226 @@
+%YAML 1.1
```

Comment:
Not sure this should be a prefab, at least not yet. Eventually for when we do it programatically I reckon it'll be better to bundle all managers into one prefab and use that in the scene.

Thoughts?